tensorflow/docs_src/programmers_guide/faq.md

```diff
@@ -149,6 +149,8 @@ TensorFlow also has a
 to help build support for more client languages.  We invite contributions of new
 language bindings.
 
+An experimental Scala API that replicates a lot of the Python API functionality using the C API is located at [eaplatanios/tensorflow_scala](https://github.com/eaplatanios/tensorflow_scala).
```

Comment:
Here too I'd like to ensure that the reader is clear about the distinction between bindings that are maintained by (and for which they can expect support for) by the TensorFlow organization and those owned, maintained and supported by others.

How about the following instead:

Bindings for various other languages (such as [C#](https://github.com/migueldeicaza/TensorFlowSharp), [Julia](https://github.com/malmaud/TensorFlow.jl), [Ruby](https://github.com/somaticio/tensorflow.rb) and [Scala](https://github.com/eaplatanios/tensorflow_scala)) created and supported by the opensource community build on top of the [C API](https://www.tensorflow.org/code/tensorflow/c/c_api.h) supported by the TensorFlow maintainers.